.circleci/config.yml

```diff
@@ -278,9 +296,17 @@ jobs:
       - set-jekyll-basename
       - trigger-docs-platform-build
 
+  deploy-preview:
+    docker:
+      - image: cibuilds/aws:1.16.185
```

Comment:
This image is one I made for my personal project awhile ago. It should work but I haven't supported it in a log time. There are official CircleCI AWS images though I believe it only supports v2 of the CLI. Just an FYI.